app/controllers/account/account_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Account
+  class AccountController < ApplicationController
```

Comment:
doesn't seem to be a useful abstraction
would suggest putting that two line to the user controller directly. or at least register a new concern(e.g. `controllable`) for shared pundit methods

Reply:
also have a look at https://github.com/varvet/pundit#rescuing-a-denied-authorization-in-rails
example: https://www.bigbinary.com/learn-rubyonrails-book/keep-application-controller-clean

Reply:
done